templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-service-guacamole-linuxvm
-version: 0.6.9
+version: 1.0.0
```

Comment:
Any reason for this big jump in version number?

Reply:
Due to it needing the new version of Nexus, so the resources can't really just be upgraded